src/theme.ts

```diff
@@ -1,8 +1,8 @@
 import {KeyPaths} from './utils/types/KeyPaths'
-import primitives from '@primer/primitives'
+import {colors} from './legacy-theme/js/colors'
 import {partitionColors, fontStack, omitScale} from './utils/theme'
 
-const {lineHeight: lineHeights} = primitives.typography.normal
+// const {lineHeight: lineHeights} = primitives.typography
```

Comment:
kinda strange that we were only using lineHeight and colors from primitives. Makes it easier for this PR I guess 😅